requirements.txt

```diff
@@ -5,6 +5,3 @@ python-multipart>=0.0.6
 black>=23.10.0
 pylint>=3.0.1
 Requests==2.31.0
-
```

Comment:
I checked your git history. It looks like you merged the main branch with your branch. Try to do rebase instead.

Reply:
you might need to do some git history rewriting. Let me know if you need help.